src/runtime/ArrayObject.h

```diff
@@ -128,9 +131,17 @@ class ArrayObject : public Object {
         return m_arrayLength;
     }
 
+    bool hasFastModeDataBuffer()
+    {
+#if defined(ESCARGOT_64) && defined(ESCARGOT_USE_32BIT_IN_64BIT)
```

Comment:
Is it possible to replace existing `isFastArrayMode` method by this new `hasFastModeDataBuffer`?

Reply:
if array length is 0 and array is length
there is no fast mode data buffer..

Reply:
I get it